app/src/main/java/org/oppia/android/app/options/OptionsActivityPresenter.kt

```diff
@@ -25,7 +25,11 @@ class OptionsActivityPresenter @Inject constructor(
     isFirstOpen: Boolean,
     selectedFragment: String
   ) {
-    activity.setContentView(R.layout.option_activity)
+    if (isFromNavigationDrawer) {
```

Comment:
This code should be merged with the `if` condition in line `39`

Reply:
@rt4914 
it will look like this similar to `helppresenter`
` if (isFromNavigationDrawer) {
      activity.setContentView(R.layout.option_activity)
      setUpToolbar()
      setUpNavigationDrawer()
    } else {
      setUpToolbar()
      activity.setContentView(R.layout.options_without_drawer_activity)
      activity.supportActionBar!!.setDisplayHomeAsUpEnabled(true)
      toolbar.setNavigationOnClickListener {
        activity.finish()
      }
    }`

Reply:
@chrk2205 The above comment is not that readable. Can you answer my suggestion is doable or not?

Reply:
it is doable 
it looks similar to in `helpActivityPresenter` ( above changed file ).

Reply:
Thanks.